x/staging/posts/types/msgs.go

```diff
@@ -386,11 +382,11 @@ func (msg MsgRegisterReaction) ValidateBasic() error {
 	}
 
 	if !commons.IsURIValid(msg.Value) {
-		return sdkerrors.Wrap(commonerrors.ErrInvalidURI, "reaction value should be a valid uri")
+		return ErrReactionInvalidValue
 	}
 
 	if !subspacestypes.IsValidSubspace(msg.Subspace) {
-		return sdkerrors.Wrap(ErrInvalidSubspace, "reaction subspace must be a valid sha-256 hash")
+		return sdkerrors.Wrap(subspacestypes.ErrInvalidSubspaceID, "reaction subspace must be a valid sha-256 hash")
```

Comment:
No need to specify why it's invalid, but we can wrap the invalid value instead
```suggestion
		return sdkerrors.Wrap(subspacestypes.ErrInvalidSubspaceID, msg.Subspace)
```